pkg/tfgen/docs.go

```diff
@@ -425,20 +418,9 @@ func parseTFMarkdown(g *generator, info tfbridge.ResourceOrDataSourceInfo, kind
 						ret.Description += "\n"
 					}
 				}
-
-				description := strings.Join(subsection, "\n") + "\n"
-				if headerIsExampleUsage {
-					// Wrap each example in shortcode.
-					description = "{{% example %}}\n" + description + "{{% /example %}}\n"
-				}
-				ret.Description += description
+				ret.Description += strings.Join(subsection, "\n") + "\n"
```

Comment:
Is this change intentional?

Reply:
@praneetloke i just hit revert on the original PR - this wasnt manually reverted

Reply:
Looks like it was done in the original PR https://github.com/pulumi/pulumi-terraform-bridge/pull/130/files#diff-9cad96fc5e12fbb60a4c1204a57f5035L421

Reply:
this was just a GH based revert - not manual

Reply:
Ah yes I see (it was moved a line below where it originally was.) Thanks for confirming, Paul!